Userland/Applications/PixelPaint/PaletteWidget.h

```diff
@@ -1,11 +1,13 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
```

Comment:
Same comment about the copyrights:
```suggestion
 * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
 * Copyright (c) 2021, Your Name <your@email>
```